apps/Gruntfile.js

```diff
@@ -404,6 +404,8 @@ module.exports = function (grunt) {
 
     pd: './src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx',
 
+    teacherApplication: './src/code-studio/pd/teacher_application/teacher_application.jsx',
```

Comment:
Good question @aoby. The entry point should be responsible for getting that data, and explicitly passing it into any library code. As for _how_ the entry point gets the data, the solution @Bjvanminnen and I landed on that we felt was the most explicit and bomb proof was this: https://github.com/code-dot-org/code-dot-org/blob/staging-next/apps/src/util/getScriptData.js

Or for the lazy, make your html look like this:
```html
<script src="myscript.js" data-someServerData='{"userId": "foobar"}'></script>
```

or your haml look like:
```haml
%script{src: minifiable_asset_path('js/myscript'),
        data: {someServerData: {userId: 'foobar'}.to_json}}
```

and make your js look like this:
```js
import getScriptData from '@cdo/apps/util/getScriptData';
const userId = getScriptData('someServerData').userId;
console.log("the server that rendered this page claims the user is", userid);
```